docs/installation/introduction.rst

```diff
@@ -14,7 +112,7 @@ Nodes
 containers and can be managed by the cluster (control plane services,
```

Comment:
I realized Node may be confusing when you think about Nodes having Node role no?

Reply:
The `node-role.kubernetes.io/node` role label doesn't really do much, and indeed, is slightly confusing. But we can't change the core K8s names at our will, so I don't see what kind of improvement could be added here.

Reply:
Yes I am not sure how to improve as well ...

Reply:
I think we can omit it entirely. I haven't heard of workloads using this `nodeSelector` anyway, as most wouldn't use one at all, and be scheduled anywhere not tainted. I'll adjust the schemas to represent "workload services" instead of the confusing "node" role.